test/verify/files/cockpitMachinesTestExternalProvider.js

```diff
@@ -173,6 +176,18 @@
     },
 
     /**
+     * Call `Send Non-Maskable Interrupt` on the VM.
+     */
+    SENDNMI_VM: function (payload) {
+      console.log('PROVIDER.SENDNMI_VM called with params: ' + JSON.stringify(payload));
```

Comment:
there's logDebug() function in helpers.es6. Debug output can be switched on/off in config.es6